doc/src/cylc-user-guide/cug.tex

```diff
@@ -1257,12 +1257,12 @@ \subsubsection{Configuring Cylc Review}
 Alternatively you can run the Cylc Review web service under Apache
 \lstinline=mod_wsgi=. To do this you will need to set up an Apache module
 configuration file (typically in
-\lstinline=/etc/httpd/conf.d/rose-wsgi.conf=) containing the following (with
+\lstinline=/etc/httpd/conf.d/cylc-wsgi.conf=) containing the following (with
 the paths set appropriately):
 
 \lstset{language=bash}
 \begin{lstlisting}
-WSGIPythonPath /path/to/rose/lib/python
+WSGIPythonPath /path/to/cylc/lib
```

Comment:
There's one sub-directory component missing now, as per @hjoliver's comment:

> so this should probably be <cylc-dir>/lib/cylc (location of review.py)

``path/to/lib/cylc`` might be more coherent than ``path/to/cylc/lib/cylc``?